src/main/java/essenmakanan/command/AddRecipeCommand.java

```diff
@@ -157,7 +157,10 @@ public void addValidRecipe() throws EssenFormatException {
             }
             slashIndex = nextSlashIndex;
         }
-
+        if (recipeTitle.length() == 0) {
```

Comment:
This works and solves #159! But is recipeTitle.isEmpty() more appropriate for strings?

Reply:
Thanks, will modify later